bin/fuel-core/src/cli/run/graphql.rs

```diff
@@ -24,6 +24,14 @@ pub struct GraphQLArgs {
     #[clap(long = "graphql-max-recursive-depth", default_value = "16", env)]
     pub graphql_max_recursive_depth: usize,
 
+    /// The max body limit of the GraphQL query.
+    #[clap(
+        long = "graphql-max-queries-body-limit",
+        default_value = "1048576",
```

Comment:
As a default, this seems a bit excessive. The built-in default axum limit is 2MB, so I'm not sure that halving that is the best default. I'd think somewhere in the 256KB range as a default, given how much data traversal (i.e., computational expensive queries) can be easily put into a single request.

Reply:
The hex representation of the `Create` transaction can be `512KB`. I increased it to `1MB` just to allow additional data in the query or dry run two heavy transactions.

Before it was unlimited=)

Reply:
Got it, makes sense. I misread that we disabled the max limit previously - in that case, 1MB seems reasonable to me.